tests/e2e/scale/noobaa/test_hsbench.py

```diff
@@ -57,3 +75,28 @@ def test_s3_benchmark_hsbench(self, hsbenchs3):
 
         # Check ceph health status
         utils.ceph_health_check()
+
+    @bugzilla("1998680")
+    @skipif_ocs_version("<4.9")
+    @pytest.mark.polarion_id("OCS-2698")
+    def test_s3_benchmark_object_bucket(self, s3bench, mcg_obj, bucket_factory):
+        """
+        Tests large number of objects on a single bucket:
+        * Create an OBC
+        * Run hs S3 benchmark to create 700k objects on the object bucket
+        * Post writing objects verify OBC creation
+        """
+        # Create an Object bucket
+        object_bucket = bucket_factory(amount=1, interface="OC")[0]
+
+        # Write 700k objects to the object bucket
+        s3bench.run_benchmark(
+            num_obj=700000,
+            timeout=12000,
+            access_key=mcg_obj.access_key_id,
+            secret_key=mcg_obj.access_key,
+            end_point=f"http://s3.openshift-storage.svc/{object_bucket.name}",
+        )
+
+        # Create new OBC and verify it is bound
+        bucket_factory(interface="OC")
```

Comment:
How new OBCs get created here?

The https://bugzilla.redhat.com/show_bug.cgi?id=1998680#c38 mentions you created 100 OBCs.

Reply:
Does not matter, OBC should be bound instead of going to LOST or PENDING state.